src/main/java/org/prebid/server/bidder/ix/IxBidder.java

```diff
@@ -243,7 +258,9 @@ private List<BidderBid> extractBids(BidResponse bidResponse, BidRequest bidReque
                 : bidsFromResponse(bidResponse, bidRequest, errors);
     }
 
-    private List<BidderBid> bidsFromResponse(BidResponse bidResponse, BidRequest bidRequest, List<BidderError> errors) {
+    private List<BidderBid> bidsFromResponse(BidResponse bidResponse,
```

Comment:
the `makeBids` method is redundant, please deprecate it

Reply:
Will do. We already discussed this with Alex.

Reply:
Actually, it helps me identify another error, that was left in IX long time ago. Also fixed.